pkg/executor/executor_test.go

```diff
@@ -108,9 +108,10 @@ func TestPrefix(t *testing.T) {
 	buf := &bytes.Buffer{}
 
 	executor := NewShell("echo \"line1\nline2\nline3\"")
-	executor.outputWriter = buf
-	executor.SetOutputPrefix("---")
-	result := executor.Run()
+	impl := executor.(*executorImpl)
+	impl.outputWriter = buf
```

Comment:
I was searching for a better way but i cannot think of something

Reply:
I think this is fine for now. We could improve this by introducing a `executor.SetOutputWriter(writer)` at some point.